docs/Formula-Cookbook.md

```diff
@@ -761,44 +761,79 @@ For example, Ruby 1.9’s gems should be installed to `var/lib/ruby/` so that ge
+```
+
+2. If the formula does not provide a service you can generate one using the following stanza.
+```rb
+service do
+  run bin/"script"
+end
 ```
 
-1. If the formula does not provide a plist you can add a plist using the following stanzas.
-This will define what the user can run manually instead of the launchd service.
+#### Service block methods
+There are many more options you can set within such a block, and in this table you will find them all.
+The only required field in a `service` block is the `run` field to indicate what to run.
+
+| Method                  | Default      | macOS | Linux | Description                                                                             |
+|-------------------------|--------------|-------|-------|-----------------------------------------------------------------------------------------|
+| `run`                   | -            |  yes  |  yes  | Command to execute, an array with arguments or a path                                   |
+| `run_type`              | `:immediate` |  yes  |  yes  | The type of service, `:immediate`, `:interval` or `:cron`                               |
+| `keep_alive`            | `false`      |  yes  |  yes  | If the service needs to keep the process running after exit                             |
+| `interval`              | -            |  yes  |  yes  | Controls the start interval, required for the `:interval` type.                         |
+| `cron`                  | -            |  yes  |  yes  | Controls the trigger times, required for the `:cron` type                               |
+| `environment_variables` | -            |  yes  |  yes  | A hash of variables to set                                                              |
+| `working_dir`           | -            |  yes  |  yes  | The directory to operate from                                                           |
+| `root_dir`              | -            |  yes  |  yes  | The directory to use as a chroot for the process                                        |
+| `input_path`            | -            |  yes  |  yes  | Path to use as input for the process                                                    |
+| `log_path`              | -            |  yes  |  yes  | Path to write stdout to                                                                 |
+| `error_log_path`        | -            |  yes  |  yes  | Path to write stderr to                                                                 |
+| `restart_delay`         | -            |  yes  |  yes  | The delay before restarting a process                                                   |
+| `process_type`          | -            |  yes  | no-op | The type of process to manage `:background`, `:standard`, `:interactive` or `:adaptive` |
+| `macos_legacy_timers`   | -            |  yes  | no-op | Timers created by launchd jobs are coalesced unless this is set                         |
```

Comment:
Include [`launch_only_once`](https://github.com/Homebrew/brew/pull/12837)? 
(The trailing pipes on each line aren't required btw.)

Reply:
In my defense, that wasn't merged when I wrote this 😄. I'll do it later